runtime/testy/stateManager.py

```diff
@@ -42,6 +44,22 @@ def addPipe(self, processName, pipe):
     self.processMapping[processName] = pipe
     pipe.send(RUNTIME_CONFIG.PIPE_READY.value)
 
+  def createKey(self, keys):
+    currDict = self.state
+    for key in keys:
+      try:
+        if key not in currDict:
+          currDict[key] = {}
+        currDict = currDict[key]
+      except TypeError:
+        error = StudentAPIKeyError(
+          "key '{}' is defined, but does not contain a dictionary.".format(key))
+        self.processMapping[PROCESS_NAMES.STUDENT_CODE].send(error)
+        return
+    # Returns new state as a string
+    message = json.dumps(self.state, indent=4)
```

Comment:
We dont want to return the state for two reasons. 
1. We dont want students having access to the entire state.
2. That's a lot of data to send, and we wont use nearly all of it.

If this error occurs, it seems like something internal in runtime broke. I think we should consider it a BadThing.

Would also mean we won't need to import the json library.